### ISSUE TITLE
Missing "show techniques and failures" controls

### ISSUE THREAD

**Issue by fstrr** [w3c/wai-wcag-quickref#254]
As [reported in the WCAG repo](https://github.com/w3c/wcag/issues/3589), there are missing "show techniques and failures" controls for:

- [1.3.5 Identify Input Purpose](https://www.w3.org/WAI/WCAG22/quickref/#identify-input-purpose)
- [2.1.3 Keyboard No Exception](https://www.w3.org/WAI/WCAG22/quickref/#keyboard-no-exception)
- [2.4.10 Section Headings](https://www.w3.org/WAI/WCAG22/quickref/#section-headings)
- [3.3.6 Error Prevention (All)](https://www.w3.org/WAI/WCAG22/quickref/#error-prevention-all)
